src/application.c

```diff
@@ -35,11 +35,16 @@ void Run() {
   printf("\n");
 
   printf("GETTING TRIP DATA...\n");
-  GetTripData(kCurl, kStartLocationInput, kStopLocationInput);
+  Trips* trips = GetTrips(kCurl, kStartLocationInput, kStopLocationInput);
```

Comment:
@SebastianMygind I saw you also edited those lines here. Can we merge my PR first, and then you can resolve the merge conflicts on your branch by using _your_ input data and passing it to `GetTrips`?

Reply:
ye